### PR TITLE
Just some rubocop fixes to make travis pass again

### PR DESCRIPTION
**Pull request by emaglio**
- Run travis for ruby 2.5.0 as well

- Run rubocop only in travis and for ruby 2.5.0

- fix rubocop_todo stuff only for the tests

### CODE REVIEW

File: lib/trailblazer/operation.rb [hunk 1]
```diff
@@ -29,8 +29,8 @@ module Trailblazer
   class Operation
     module FastTrackActivity
       builder_options = {
-        track_end:     Railway::End::Success.new(semantic: :success),
-        failure_end:   Railway::End::Failure.new(semantic: :failure),
+        track_end: Railway::End::Success.new(semantic: :success),
+        failure_end: Railway::End::Failure.new(semantic: :failure),
```
Comment (apotonick) [inline lib/trailblazer/operation.rb]: Not a fan of this, looks messy.

Comment (konung) [reply lib/trailblazer/operation.rb]: The setting for rubocop to stick to the other style of formatting (i.e. align on colon - which I also prefer) is this:

```yaml
Layout/AlignHash:
  EnforcedColonStyle: separator
```

Comment (emaglio) [reply lib/trailblazer/operation.rb]: @konung :+1: thanks

**Comment by emaglio** [comment]
we are going to remove rubocop - waste of time!

**Comment by fernandes** [comment]
🙌 🙌 🙌

**Pull request closed without merging**